platforms/platform_db/platform_defaults.json

```diff
@@ -59,10 +48,106 @@
                      "is the virtual channel number.)  Maximum bandwidth is typically a",
                      "function of the number lines in flight, indepent of request sizes."
                   ],
-               "max-bw-active-lines-c0": "{ 768, 512, 256, 256 }",
+               "max-bw-active-lines-c0": "{ 512, 512, 256, 256 }",
                "max-bw-active-lines-c1": "{ 512, 256, 128, 128 }",
 
-               "max-outstanding-mmio-rd-reqs": 32
+               "max-outstanding-mmio-rd-reqs": 32,
+
+               "comment":
+                  [
+                     "CCI-P requires that an AFU register both the request and response",
+                     "signals before any combinational logic.  On some platforms, it",
+                     "is easier to meet timing when there is more than one register",
+                     "stage.  num-timing-reg-stages is a platform-specific recommendation",
+                     "to AFUs, suggesting the number of registered CCI-P request and",
+                     "response stages for timing across the partial reconfiguration boundary.",
+                     "",
+                     "Note: There is no extra platform-side buffering added for handling",
```

Comment:
I dont understand the explanation of "almost full" here. Can you clarify?

Reply:
Buffering delays the AFU receiving almost full. During that delay it may be sending requests, which count against the almost full limit.

Reply:
Does this comment mean anything to the AFU developer other than that they should pay attention to almost full when  registered stages are present?

Reply:
No, that's it.